install-dev/langs/bn/data/country.xml

```diff
@@ -1,735 +1,777 @@
+    <name>&#2475;&#2495;&#2482;&#2495;&#2474;&#2494;&#2439;&#2472;</name>
+  </country>
+  <country id="PK">
+    <name>&#2474;&#2494;&#2453;&#2495;&#2488;&#2509;&#2468;&#2494;&#2472;</name>
+  </country>
+  <country id="PL">
+    <name>&#2474;&#2507;&#2482;&#2509;&#2479;&#2494;&#2472;&#2509;&#2465;</name>
+  </country>
+  <country id="PM">
+    <name>&#2488;&#2503;&#2472;&#2509;&#2463; &#2474;&#2495;&#2479;&#2492;&#2503;&#2480; &#2451; &#2478;&#2495;&#2453;&#2497;&#2479;&#2492;&#2503;&#2482;&#2472;</name>
   </country>
   <country id="PN">
-    <name>Pitcairn</name>
+    <name>&#2474;&#2495;&#2463;&#2453;&#2503;&#2479;&#2492;&#2494;&#2480;&#2509;&#2472; &#2470;&#2509;&#2476;&#2496;&#2474;&#2474;&#2497;&#2462;&#2509;&#2460;</name>
   </country>
   <country id="PR">
-    <name>Puerto Rico</name>
+    <name>&#2474;&#2497;&#2479;&#2492;&#2503;&#2480;&#2509;&#2468;&#2507; &#2480;&#2495;&#2453;&#2507;</name>
+  </country>
+  <country id="PS">
+    <name>&#2475;&#2495;&#2482;&#2495;&#2488;&#2509;&#2468;&#2495;&#2472; &#2437;&#2462;&#2509;&#2458;&#2482;&#2488;&#2478;&#2498;&#2489;</name>
+  </country>
+  <country id="PT">
+    <name>&#2474;&#2480;&#2509;&#2468;&#2497;&#2455;&#2494;&#2482;</name>
+  </country>
+  <country id="PW">
+    <name>&#2474;&#2494;&#2482;&#2494;&#2441;</name>
+  </country>
+  <country id="PY">
+    <name>&#2474;&#2509;&#2479;&#2494;&#2480;&#2494;&#2455;&#2497;&#2479;&#2492;&#2503;</name>
   </country>
   <country id="QA">
-    <name>Qatar</name>
+    <name>&#2453;&#2494;&#2468;&#2494;&#2480;</name>
   </country>
-  <country id="RE">
-    <name>Reunion Island</name>
+  <country id="QO">
+    <name>&#2438;&#2441;&#2463;&#2482;&#2494;&#2439;&#2472;&#2495;&#2434; &#2451;&#2488;&#2494;&#2472;&#2495;&#2479;&#2492;&#2494;</name>
   </country>
-  <country id="RU">
-    <name>Russian Federation</name>
+  <country id="RE">
+    <name>&#2480;&#2495;&#2439;&#2441;&#2472;&#2495;&#2479;&#2492;&#2472;</name>
   </country>
-  <country id="RW">
-    <name>Rwanda</name>
+  <country id="RO">
+    <name>&#2480;&#2497;&#2478;&#2494;&#2472;&#2495;&#2479;&#2492;&#2494;</name>
   </country>
-  <country id="BL">
-    <name>Saint Barthelemy</name>
+  <country id="RS">
+    <name>&#2488;&#2494;&#2480;&#2509;&#2476;&#2495;&#2479;&#2492;&#2494;</name>
   </country>
-  <country id="KN">
-    <name>Saint Kitts and Nevis</name>
+  <country id="RU">
+    <name>&#2480;&#2494;&#2486;&#2495;&#2479;&#2492;&#2494;</name>
   </country>
-  <country id="LC">
-    <name>Saint Lucia</name>
+  <country id="RW">
+    <name>&#2480;&#2497;&#2479;&#2492;&#2494;&#2472;&#2509;&#2465;&#2494;</name>
   </country>
-  <country id="MF">
-    <name>Saint Martin</name>
+  <country id="SA">
+    <name>&#2488;&#2508;&#2470;&#2495; &#2438;&#2480;&#2476;</name>
   </country>
-  <country id="PM">
-    <name>Saint Pierre and Miquelon</name>
+  <country id="SB">
+    <name>&#2488;&#2482;&#2507;&#2478;&#2472; &#2470;&#2509;&#2476;&#2496;&#2474;&#2474;&#2497;&#2462;&#2509;&#2460;</name>
```

Comment:
Cool!

Did some research and it all the characters can be represented with `UTF-8`